packages/db/tests/translate/test-async-operations.js

```diff
@@ -67,3 +141,31 @@ test('translateApiAsyncOperationToPostgresAsyncOperation discards \'none\' outpu
 
   t.deepEqual(translateApiAsyncOperationToPostgresAsyncOperation(apiAsyncOperation), expected);
 });
+
+test('translatePostgresAsyncOperationToApiAsyncOperation translates PostgreSQL record to async operation API record', async (t) => {
+  const id = randomString();
+  const taskArn = randomString();
+  const createdAt = new Date();
+  const updatedAt = new Date();
+  const description = 'Some async run';
+  const pgAsyncOperation = fakeAsyncOperationRecordFactory({
+    id,
+    task_arn: taskArn,
+    created_at: createdAt,
+    updated_at: updatedAt,
+    description,
+  });
+
+  const expectedAsyncOperation = {
+    id,
+    status: 'RUNNING',
+    taskArn,
+    description,
+    operationType: 'ES Index',
+    output: JSON.stringify({ test: 'output' }),
```

Comment:
We should be passing some sort of object into `fakeAsyncOperationRecordFactory` for the `output` so that we can be certain our assertion will pass. Otherwise the default in `fakeAsyncOperationRecordFactory` could change and this test would fail

Reply:
[`44b3862` (#2294)](https://github.com/nasa/cumulus/pull/2294/commits/44b3862da93c36df3cc7b7e8034248a42d2f0b44)